protostar/commands/test/cheatcodes/reflect/cairo_struct.py

```diff
@@ -0,0 +1,80 @@
+from typing import OrderedDict, Union, Any, List, Tuple
+from starkware.cairo.lang.vm.relocatable import RelocatableValue
+from protostar.commands.test.test_environment_exceptions import CheatcodeException
+from protostar.starknet.hint_local import HintLocal
+
+
+class CairoStruct:
+    def __init__(self, *args, **kwargs) -> None:
+        if len(args) > 0:  # / syntax not avaliable in py3.7
+            raise CheatcodeException(
```

Comment:
We should use SimpleReportedException here instead of CheatcodeException, CairoStruct is not strictly tied to reflect

Reply:
I don't really share the opinion that it is not strictly tied. Why do you think so?